src/components/Card/NewCard.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MouseEventHandler, useState } from 'react';
+import styles from './NewCard.module.css';
+
+// components
+import Button from '../UI/Button';
+import Card from '../UI/CardWrapper';
+import DeleteIcon from '../UI/Icons/DeleteIcon';
+import Input from '../UI/Input';
+
+interface NewCardProps {
+  onCancelNewCard: MouseEventHandler;
+  onSaveNewCard: Function;
+}
+
+const NewCard: React.FC<NewCardProps> = (props) => {
+  const [formStep, setFormStep] = useState<number>(1);
```

Comment:
**nitpick:** For better readability, I would strongly suggest to use enum instead of int, as it will be easier to maintain sides here. Eg:

```ts
enum CardSide {
  front,
  back
}
```

then: 

```tsx
  const [formStep, setFormStep] = useState<CardSide>(CardSide.front);
```